homeassistant/components/zwave/__init__.py

```diff
@@ -775,8 +808,6 @@ def _check_entity_ready(self):
             node_config.get(CONF_POLLING_INTENSITY), int)
         if polling_intensity:
             self.primary.enable_poll(polling_intensity)
-        else:
-            self.primary.disable_poll()
```

Comment:
Question: Is polling disabled by default, and that's why this was removed?

Reply:
Polling is always set to 0 by OZW unless it is set to something different by user to the value_id.
So if there is no entry for polling, in either the homeassistant config or the zwcfg, no polling is set.

Reply:
Ref: https://github.com/OpenZWave/open-zwave/blob/master/cpp/src/value_classes/Value.cpp#L123

Reply:
:+1: Just checking.

Reply:
Appreciate it :1st_place_medal: